### PR TITLE
redirect upon government form submit

### LINKED ISSUE #3104: Government Information Form not redirected to profile page after submitting the form
https://user-images.githubusercontent.com/95497896/174168690-e0df83c6-de3e-4dd1-997e-610552209b2b.mp4

### PR DESCRIPTION
**Pull request by vd1992**
resolves #3104 

testing: ensure redirect occurs upon submitting the government info form

### CODE REVIEW

File: frontend/talentsearch/src/js/components/GovernmentInfoForm/GovernmentInfoForm.tsx [hunk 1]
```diff
@@ -495,9 +495,13 @@ export const GovInfoFormContainer: React.FunctionComponent = () => {
           const message = intl.formatMessage(profileMessages.profileCompleted);
           if (!preProfileStatus && currentProfileStatus) {
             toast.success(message);
+            navigate(paths.profile());
           }
           navigate(paths.profile());
           toast.success(intl.formatMessage(profileMessages.userUpdated));
+        } else {
+          navigate(paths.profile());
+          toast.success(intl.formatMessage(profileMessages.userUpdated));
```
Comment (esizer) [inline frontend/talentsearch/src/js/components/GovernmentInfoForm/GovernmentInfoForm.tsx:504]: Is it possible to move the `toast` and `navigate` just after the first `if` statement to reduce the duplication occurring and no longer require an `else` statement?

**Review by esizer — CHANGES_REQUESTED** [review]
Just one small request

File: frontend/talentsearch/src/js/components/GovernmentInfoForm/GovernmentInfoForm.tsx (hunk 1 above)
Comment (vd1992) [reply frontend/talentsearch/src/js/components/GovernmentInfoForm/GovernmentInfoForm.tsx]: [like so?](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3175/commits/ac9605049024064029a72ffef3d686b634c0abc1)

**Review by esizer — APPROVED** [review]
Very nice! 😄

**Pull request merged**